modules/open-url/open-url.ts

```diff
@@ -13,7 +15,29 @@ function genericOpen(url: string): Promise<boolean> {
 		})
 }
 
-export function openUrl(url: string): Promise<boolean> {
+async function launchBrowser(url: string): Promise<boolean> {
+	try {
+		if (await InAppBrowser.isAvailable()) {
+			await InAppBrowser.open(url, {
+				animated: true,
+				showTitle: true,
+				enableUrlBarHiding: true,
+				enableDefaultShare: true,
+				modalPresentationStyle: 'currentContext',
+			})
+		} else {
+			// fall back to opening in Chrome / Browser / platform default
+			genericOpen(url)
+		}
```

Comment:
Does this read cleanly enough? I'm also thinking about this variation for the `try` block:

```ts
try {
    const canOpenCustomBrowser = await InAppBrowser.isAvailable()

    if (canOpenCustomBrowser === false) {
      // fall back to opening in Chrome / Browser / platform default
      genericOpen(url)
      return true
    }

    await InAppBrowser.open(url, {
      animated: true,
      showTitle: true,
      enableUrlBarHiding: true,
      enableDefaultShare: true,
      modalPresentationStyle: 'currentContext',
    })
} catch (error) {
    ...
}
```

Reply:
The current implementation may be easier to follow if less returns were added.